README.md

```diff
@@ -410,7 +410,8 @@ In order to set these configurations manually, edit `demo_config.h` in `demos/mq
 * Set `CLIENT_CERT_PATH` to the path of the client certificate downloaded when setting up the device certificate in [AWS IoT Account Setup](#aws-iot-account-setup).
 * Set `CLIENT_PRIVATE_KEY_PATH` to the path of the private key downloaded when setting up the device certificate in [AWS IoT Account Setup](#aws-iot-account-setup).
 
-It is possible to configure `ROOT_CA_CERT_PATH` to any PEM-encoded Root CA Certificate. However, this is optional because CMake will download and set it to [AmazonRootCA1.pem](https://www.amazontrust.com/repository/AmazonRootCA1.pem) when unspecified.
+It is possible to configure `ROOT_CA_CERT_PATH` to any PEM-encoded Root CA Certificate. However, this is optional because CMake will download and set it to [AmazonRootCA1.pem](https://www.amazontrust.com/repository/AmazonRootCA1.pem) when unspecified. If unspecified, the default Root CA
```

Comment:
Should split this line up so that it's not a single line that's 300 characters wide

Reply:
I think doing that will make this single paragraph inconsistent from the whole readme. The whole readme is written with paragraphs on a single line.

Reply:
I agree with your point Aniruddha. We may want to rework this README later. I'd prefer not to add to the problem.

Also - splitting these on separate lines doesn't lead to any changes in how the markdown is rendered, so I am fine splitting.